src/P3_terminal_velocity.jl

```diff
@@ -95,134 +83,99 @@ cloud or rain drop as a function of their sizes. It uses Chen 2022 velocity
 parameterization for ice and rain and assumes no sedimentation of cloud droplets.
 """
 function velocity_difference(
-    pdf_r::Union{
+    ::Union{
         CMP.RainParticlePDF_SB2006{FT},
         CMP.RainParticlePDF_SB2006_limited{FT},
     },
     Dₗ::FT,
     Dᵢ::FT,
-    p3::PSP3,
+    state::P3State,
     Chen2022::CMP.Chen2022VelType,
     ρₐ::FT,
-    F_rim::FT,
-    th,
     use_aspect_ratio = true,
 ) where {FT}
     # velocity difference for rain-ice collisions
     return abs(
         p3_particle_terminal_velocity(
-            p3,
+            state,
             Dᵢ,
             Chen2022,
             ρₐ,
-            F_rim,
-            F_liq,
-            th,
             use_aspect_ratio,
         ) - CM2.rain_particle_terminal_velocity(Dₗ, Chen2022.rain, ρₐ),
     )
 end
 function velocity_difference(
-    pdf_c::CMP.CloudParticlePDF_SB2006{FT},
+    ::CMP.CloudParticlePDF_SB2006{FT},
     Dₗ::FT,
     Dᵢ::FT,
-    p3::PSP3,
+    state::P3State,
     Chen2022::CMP.Chen2022VelType,
     ρₐ::FT,
-    F_rim::FT,
-    F_liq::FT,
-    th,
     use_aspect_ratio = true,
 ) where {FT}
     # velocity difference for cloud-ice collisions
     return abs(
         p3_particle_terminal_velocity(
-            p3,
+            state,
```

Comment:
Actually, this should now use the sedimentation terminal velocity for cloud liquid. You can fix it now or open an issue and we can do it separately